Can't build

I'm getting:
```
make: *** No rule to make target 'native/lib.c', needed by '/home/olafura/dev/pelemay/_build/dev/lib/pelemay/priv/libnif.so'. Stop.
** (Mix) Could not compile with "make" (exit status: 2).
You need to have gcc and make installed. If you are using
Ubuntu or any other Debian-based system, install the packages
"build-essential". Also install "erlang-dev" package if not
included in your Erlang/OTP version. If you're on Fedora, run
"dnf group install 'Development Tools'".
```

--- a/Makefile
+++ b/Makefile
@@ -23,7 +23,7 @@ endif
 calling_from_make:
 	mix compile
 
-.PHONY: all libnifvec clean
+.PHONY: all libnif clean
 
 all: $(BUILD) $(PREFIX) $(PREFIX)/libnif.so
 

--- a/mix.exs
+++ b/mix.exs
@@ -6,7 +6,7 @@ defmodule Pelemay.MixProject do
       app: :pelemay,
       version: "0.0.1",
       elixir: "~> 1.9",
-      compilers: [:elixir_make | Mix.compilers()],
+      compilers: Mix.compilers() ++ [:elixir_make],
       make_targets: ["all"],
       make_clean: ["clean"],
       start_permanent: Mix.env() == :prod,

